content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service--Deployment_Guide/Basic_Setup_of_YubiHSM_2_and_HGS.adoc

```diff
@@ -3,13 +3,13 @@
 
 === Install and Configuring YubiHSM 2
 
-Install and configure the YubiHSM 2 and software using the instructions in the following sections in the link:../YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/[YubiHSM 2 Windows Deployment Guide: Configure YubiHSM 2 Key Storage Provider for Microsoft Windows Server]:
+Install and configure the YubiHSM 2 and software using the instructions in the following sections in the link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server[YubiHSM 2 Windows Deployment Guide]:
 
-1. link:../YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Install_the_YubiHSM_Tools_and_Software.adoc[Install YubiHSM 2 Tools and Software]
+1. link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Install_the_YubiHSM_Tools_and_Software.adoc[Install YubiHSM 2 Tools and Software]
 
-2. link:../YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Configure_the_Primary_YubiHSM_2_Device.adoc[Configure the Primary YubiHSM 2 Device]
+2. link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Configure_the_Primary_YubiHSM_2_Device.adoc[Configure the Primary YubiHSM 2 Device]
 
-3. link:../YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Configure_the_YubiHSM_2_Software.adoc/[Configure the YubiHSM 2 Software]
+3. link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Configure_the_YubiHSM_2_Software.adoc/[Configure the YubiHSM 2 Software]
```

Comment:
There is a trailing / here after .adoc that breaks the link

Reply:
https://github.com/Yubico/developers.yubico.com/pull/249/commits/cddaa4dd34b91abc2fe5d929c77e68dd438bed26